types/circular-json/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+	"extends": "dtslint/dt.json",
+	"rules": {
+		"dt-header": false,
+		"interface-name": false
```

Comment:
`ICircularJson` was never exposed so it would be backwards-compatible to remove the `I`.